--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
@@ -403,6 +_,7 @@
         this.structureManager = new CraftStructureManager(console.getStructureManager(), console.registryAccess());
         this.serverTickManager = new CraftServerTickManager(console.tickRateManager());
         this.serverLinks = new CraftServerLinks(console);
+        cloud.victus.engine.VictusEngine.init(); // Victus - load victus.yml at startup
 
         Bukkit.setServer(this);
         // Paper start
